src/main/java/uk/nhs/tis/sync/job/PostFundingSyncJobTemplate.java

```diff
@@ -0,0 +1,75 @@
+package uk.nhs.tis.sync.job;
+
+import static uk.nhs.tis.sync.job.TrustAdminSyncJobTemplate.LAST_ENTITY_ID;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Query;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
+import uk.nhs.tis.sync.model.EntityData;
+
+/**
+ * Abstract template for post funding sync job which sets post funding status
+ *
+ * @param <T> the type of entity passed to target
+ */
+@Slf4j
+public abstract class PostFundingSyncJobTemplate<T> extends CommonSyncJobTemplate<T> {
```

Comment:
This is so specific, I can't imagine there being another `PostFundingSyncJob`.  Can we get rid of this class for now & inherit the `CommonSyncJob`?

Reply:
Ok doing it now